crates/matrix-sdk/src/sliding_sync/mod.rs

```diff
@@ -629,17 +621,17 @@ impl SlidingSync {
     /// return `Err(…)`. An `Err` will _always_ lead to the `Stream`
     /// termination.
     #[allow(unknown_lints, clippy::let_with_type_underscore)] // triggered by instrument macro
-    #[instrument(name = "sync_stream", skip_all)]
+    #[instrument(name = "sync_stream", skip_all, fields(conn_id = self.inner.id, with_e2ee = self.is_e2ee_enabled()))]
```

Comment:
We should likely add the `self.inner.position` to the fields field as well, since we removed it from the individual `debug!()` calls.

Reply:
I don't think we should include the `position` at this level: those statements are about the stream itself, and the `pos` is already shown in functions called under this span. So if we displayed it in the context too, that would display it twice. Is it ok to keep it at the `sync_once` level instead?

Reply:
Ah ok, missed that fact. Makes sense then to remove it.